### DISCUSSION TITLE
Port Quark's Witch Hat into Reliquary's

### DISCUSSION THREAD

**Discussion by xJon** [ACGaming/UniversalTweaks#450] [Tweak Suggestions]
Both Reliquary and Quark offer Witch Hats, with very similar models, although I believe Quark's is superior.
As a modpack maker, it'd be beneficial to match Reliquary's to Quark's, so either could be chosen without sacrificing aesthetics.

![javaw_4CXu6vbyXk](https://github.com/ACGaming/UniversalTweaks/assets/8945347/f6212128-65fa-4e96-a1fc-a19a8130ff45)
(Quark)

![javaw_vzLSx1LaBC](https://github.com/ACGaming/UniversalTweaks/assets/8945347/643a5090-e242-4017-bfb3-072fad5556bf)
(Reliquary)

**Comment by ACGaming** [comment]
Closed in favor or https://github.com/Elite-Modding-Team/Reliquary.
